src/System.ComponentModel.TypeConverter/src/System/ComponentModel/DateTimeConverter.cs

```diff
@@ -118,6 +121,33 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
                 return dt.ToString(format, CultureInfo.CurrentCulture);
             }
 
+            if (destinationType == typeof(InstanceDescriptor) && value is DateTime) 
+            {
+                DateTime dt = (DateTime)value;
```

Comment:
Declare inline when checking if the value is DateTime

`value is DateTime dt`

Reply:
There can't be two inline declarations of the same name.  That's why I took it out and put it where it's at.  

I can name them differently and put them inline.  Does that work?

Reply:
Ah yeah right. Nah that is fine. Could you just address the other comments?

Reply:
Yep - no prob.